source/documentation/deploying_services/redis.md

```diff
@@ -193,34 +195,34 @@ Type `yes` when asked for confirmation.
 
-### Redis plans
+### Redis service plans
+
+There are two service plans currently available for the Redis service:
 
-There are two plans available for the Redis service:
+- `tiny-3.2`
+- `medium-ha-3.2`
 
-- `tiny-clustered-3.2`
-- `tiny-unclustered-3.2`
+Do not use the `tiny-clustered-3.2` service plan for new service instances as we have deprecated this plan.
 
-You cannot migrate a service from an unclustered to a clustered plan or vice versa.
+Both service plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
 
-Both plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
+Amazon ElastiCache Redis backs up both service plans every day.
 
-#### tiny-clustered-3.2
+You can [vertically scale](/managing_apps.html#scaling) or upgrade your service plan.
 
-We recommend that you use this plan as it is backed up every day. Note that:
+#### Billing
 
-- [Sidekiq](https://sidekiq.org) [external link] does not work with clustered Redis
-- this plan cannot be [vertically scaled](/managing_apps.html#scaling) or upgraded to a bigger plan
+If your org is in its trial period, you can use the `tiny-3.2` service plan for free.
 
-#### tiny-unclustered-3.2
+Trial orgs cannot access paid service plans by default. If paid service plans are not enabled, when you try to use a paid service plan, you will receive an error stating “service instance cannot be created because paid service plans are not allowed”.
 
-Use this plan if your client library cannot connect to clustered plans. Note that:
+One of your Org Managers must contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to request that we enable paid service plans.
 
-- it is not automatically backed up
-- it can be [vertically scaled](/managing_apps.html#scaling) or upgraded to a bigger plan
+#### High availability
 
-Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you need an unclustered plan with backups enabled, or a larger unclustered plan.
+You can use a high availability service plan (`HA`). When you use a high availability service plan, Amazon ElastiCache Redis provides a hot standby service for [failover](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/AutoFailover.html) [external link] in the event that the original service instance fails.
 
-Refer to the [Redis documentation](https://redis.io/topics/cluster-tutorial) [external link] for more information on clusters.
+If you have a non-HA service plan, you will lose data during a service instance failure. Before deciding which service plan to use, you should assess your data and what type of plan you need.
```

Comment:
```suggestion
If you don't have a high availability service plan, you will lose data during a service instance failure. Before deciding which service plan to use, you should assess your data and what type of plan you need.
```
To remove the need for abbreviation and (hopefully) make things clear in the absence of the abbreviation.